twoliter/embedded/build.Dockerfile

```diff
@@ -109,7 +109,8 @@ RUN --mount=target=/host \
     rm /bypass
 
 USER builder
-RUN --mount=source=.cargo,target=/home/builder/.cargo \
+RUN --mount=source=.cargo/config.toml,target=/home/builder/rpmbuild/.cargo/config.toml \
+    --mount=source=.cargo/vendor,target=/home/builder/rpmbuild/.cargo/vendor \
```

Comment:
Why not mount the whole `.cargo` directory? Is there content in there that we're trying to keep the SDK copy of `cargo` from seeing?

Reply:
If we mount the whole .cargo directory we'll be back to square one with the error we are hitting. the .registry folder takes precedence over vendor

Reply:
and also the internal folders in the .cargo are the cargo version specific folders and files. I chose to put vendor inside .cargo root for cleanliness and to prevent .gitignore clashes.

Reply:
> If we mount the whole .cargo directory we'll be back to square one with the error we are hitting. the .registry folder takes precedence over vendor

Can you add a comment to this effect? It's valuable context to avoid an otherwise tempting refactor to simplify down to the one mount.